src/pages/HomePage.js

```diff
@@ -9,11 +9,15 @@ const HomePage = () => {
     const auth = useAuth()
     const navigate = useNavigate()
 
+    
     useEffect(()=>{
+
         if(!auth.isLoggedIn){
```

Comment:
if there is one statement in clock then no need of curly brackets and we can write in single line